src/js/api/services/StockMovementApi.js

```diff
@@ -4,11 +4,19 @@ import {
   STOCK_MOVEMENT_ROLLBACK_APPROVAL,
   STOCK_MOVEMENT_UPDATE_STATUS,
 } from 'api/urls';
+import RequisitionStatus from 'consts/requisitionStatus';
 import apiClient from 'utils/apiClient';
 
 export default {
   getStockMovements: config => apiClient.get(STOCK_MOVEMENT_API, config),
   deleteStockMovement: id => apiClient.delete(STOCK_MOVEMENT_DELETE(id)),
   updateStatus: (id, status) => apiClient.post(STOCK_MOVEMENT_UPDATE_STATUS(id), { status }),
+  rejectRequest: (id, sender, recipient, comment) =>
```

Comment:
let's do it using `({ id, sender, recipient, comment })` convention, so that you don't have to remember about the order of those arguments which might be annoying.